.github/workflows/checks.yml

```diff
@@ -46,6 +46,40 @@ jobs:
 
         exit $ok
 
+  output-regenerated:
+    name: Output Regenerated
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Set up PHP
+      uses: shivammathur/setup-php@1.8.1
+      with:
+        php-version: 7.4
+        coverage: none
+
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Download
+      run: |
+        composer update --no-interaction --prefer-dist --optimize-autoloader --prefer-stable
+
+    - name: Regenerate
+      run: |
+        ./generate regenerate S3 --all
+        ./generate regenerate Ses --all
+        ./generate regenerate Sts --all
+        ./generate regenerate Sqs --all
+
+    - name: Fix cs
+      uses: docker://oskarstark/php-cs-fixer-ga
+
+    - name: Assert up-to-date
+      run: |
+        if [ -z "$(git status src --porcelain)" ]; then exit 0; fi
+
+        git diff src
+        exit 1
```

Comment:
This will always fail

Reply:
yes. That's the purpose. It exist with 0 at line 79 when every thing is OK.

I'll inverse the IF block

Reply:
Ah, okey. I see.